server/README.md

```diff
@@ -1,5 +1,56 @@
 # Server
 
+## Google Cloud Projects
+
+**Organization:** [`whocoronavirus.org`](https://console.cloud.google.com/iam-admin/settings?organizationId=532343229286)
+
+### [`who-myhealth-staging`](https://console.cloud.google.com/home/dashboard?project=who-myhealth-staging)
+
+Domain: `staging.whocoronavirus.org`
+
+Services:
+
+- Staging App Engine
+- Staging Cloud Datastore
+- Staging FCM
```

Comment:
It might be clearer to replace "FCM" with "Firebase Project". We use Firebase for certain other services too in addition to FCM (analytics, crashlytics, App Distribution).

Reply:
```suggestion
- Staging Firebase Project
```